lib/main.dart

```diff
@@ -34,14 +36,22 @@ void main() async {
     });
   }
   if (globals.testingActive) {
-    print("This is a testing environment!");
+    String deleteStorageFlag = dotenv.env['DELETE_TEST_STORAGE'] ?? 'false';
+    globals.testingDeleteStorage = deleteStorageFlag.toBoolean();
   }
   SystemChrome.setPreferredOrientations(
       [DeviceOrientation.portraitUp, DeviceOrientation.portraitDown]);
 
   ApiDatabase apiDatabase = Locator.instance<ApiDatabase>();
   await apiDatabase.openDatabase();
-
+  if (globals.testingActive) {
```

Comment:
You can move this to `test_utils.dart` creating a function called `teardownTest`. It can be executed after each test.